lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -261,22 +263,22 @@ class TestFieldOverrideSplitPerformance(FieldOverridePerformanceTestCase):
     __test__ = True
 
     TEST_DATA = {
-        ('no_overrides', 1, True, False): (47, 1, 4, 9),
-        ('no_overrides', 2, True, False): (119, 16, 19, 54),
-        ('no_overrides', 3, True, False): (399, 81, 84, 215),
-        ('ccx', 1, True, False): (47, 1, 4, 9),
-        ('ccx', 2, True, False): (119, 16, 19, 54),
-        ('ccx', 3, True, False): (399, 81, 84, 215),
-        ('ccx', 1, True, True): (49, 1, 4, 13),
-        ('ccx', 2, True, True): (121, 16, 19, 84),
-        ('ccx', 3, True, True): (401, 81, 84, 335),
-        ('no_overrides', 1, False, False): (47, 1, 4, 9),
-        ('no_overrides', 2, False, False): (119, 16, 19, 54),
-        ('no_overrides', 3, False, False): (399, 81, 84, 215),
-        ('ccx', 1, False, False): (47, 1, 4, 9),
-        ('ccx', 2, False, False): (119, 16, 19, 54),
-        ('ccx', 3, False, False): (399, 81, 84, 215),
-        ('ccx', 1, False, True): (47, 1, 4, 9),
-        ('ccx', 2, False, True): (119, 16, 19, 54),
-        ('ccx', 3, False, True): (399, 81, 84, 215),
+        ('no_overrides', 1, True, False): (34, 0, 4, 1),
```

Comment:
All this test data is a little bit opaque. The preexisting comment explains what these fields represent, but doesn't necessarily help someone new to this code understand how they are used.